lgc/patch/PatchCopyShader.cpp

```diff
@@ -41,6 +41,7 @@
 #include "llvm/IR/IntrinsicsAMDGPU.h"
 #include "llvm/Support/CommandLine.h"
 #include "llvm/Support/Debug.h"
+#include <set>
```

Comment:
Since you are using DenseMap. Not sure if we could replace std::set with DenseSet as well.

Reply:
The comparision function required by DenseSet/DenseMap is not reloaded in `InOutLocationInfo`. I think we can make a furhter patch to do the replacement. What do you think?